src/converter/entities.ts

```diff
@@ -41,9 +41,9 @@ export const convertComputer = (
         displayName: data.hostname,
         hostname: normalizeHostname(data.hostname),
         function: ['endpoint-protection', 'anti-malware'],
-        hardwareId: data.mac_hardware_id || data.windows_processor_id,
-        installedOn: getTime(data.install_date),
-        lastSeenOn: getTime(data.last_seen),
+        hardwareId: data.windows_processor_id,
+        installedOn: parseTimePropertyValue(data.install_date),
+        lastSeenOn: parseTimePropertyValue(data.last_seen),
         installDate: undefined,
         lastSeen: undefined,
```

Comment:
I know it's outside your changes, but I'm curious if you've got any insight into the difference between `lastSeen` and the above `lastSeenOn` property.

Reply:
It just looks to be left over from one of the first commits 2 years ago. I'll remove this with the next round of PR feedback.